src/wl-copy.c

```diff
@@ -450,7 +448,7 @@ int main(int argc, char * const argv[]) {
     if (!clear) {
         if (optind < argc) {
             // copy our command-line args
-            data_to_copy = &argv[optind];
+            data_to_copy = argv[optind++];
```

Comment:
No; `data_to_copy` is of type `char * const *` (i.e. a pointer to an array of strings), not a single string.

`wl-copy` behaves like `echo` in that it supports multiple args and auto-inserts spaces between them, e.g. `wl-copy hello world` will copy the string `hello world`